hazelcast/src/main/java/com/hazelcast/nio/NodeIOService.java

```diff
@@ -239,6 +239,11 @@ public int getConnectionMonitorMaxFaults() {
     }
 
     @Override
+    public int getHandlerMigrationIntervalSeconds() {
+        return node.groupProperties.HANDLER_MIGRATION_INTERVAL_SECONDS.getInteger();
```

Comment:
yeah, it's read when a member is starting only. I followed the same pattern as other I/O components.